src/types.ts

```diff
@@ -92,51 +92,30 @@ export interface KaboomCtx {
 	 * const label = make([
 	 *     text("oh hi"),
 	 * ])
-
-	 * add([
-	 *     rect(label.width, label.height),
-	 *     color(0, 0, 255),
-	 *     children(label),
-	 * ])
+	 * 
+	 * add(label);
```

Comment:
this part is to have an example text with background which is made possible

Reply:
Any update @slmjkdbtl

Reply:
yeah something like

```js
const label = make([
    text("hi")
])

const background = add([
    rect(label.width, label.height)
])

background.add(label)
```

which might be a more practical example